deploy/helm/kubecf/assets/operations/instance_groups/api.yaml

```diff
@@ -231,11 +225,12 @@
           readiness:
             httpGet:
               port: 4002
+              scheme: HTTPS
     post_start:
       condition:
         exec:
           # policy-server doesn't support HTTP HEAD requests
-          command: [curl, --fail, --silent, http://localhost:4002/]
+          command: [curl, -k, --fail, --silent, https://localhost:4002/]
```

Comment:
[nit] Use the long version instead: `--insecure`. Whenever possible, use the long version for the command line flags.